docs/manuals/inrichting-zaakafhandelcomponent/inrichting-zaakafhandelcomponent.md

```diff
@@ -67,7 +67,11 @@ In sommige taakbehandelformulieren komen keuzelijsten voor waarvan de opties via
     ![image](images/33fc1158-0ada-4fea-a491-fc1e6e9935a0.png)
 9. Ga verder naar 'SmartDocuments'. Hier kunt u het documenttype selecteren dat door elke sjabloon moet worden verwerkt.
```

Comment:
```suggestion
9. Ga verder naar 'SmartDocuments'. Hier kunt u het documenttype selecteren dat door elk sjabloon moet worden verwerkt.
```

This sentence is not quite right I would say. Maybe @camielb can improve it further. :-)